tensorflow_lite_support/c/task/processor/segmentation_result.cc

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow_lite_support/c/task/processor/segmentation_result.h"
+
+#include <cstdlib>
+#include <memory>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+void TfLiteSegmentationResultDelete(
```

Comment:
Shouldn't also confidence_masks or category_mask be deleted here?

Reply:
Fixed